packages/skeleton-loader/src/CardSkeleton/CardSkeleton.stories.tsx

```diff
@@ -1,24 +1,32 @@
 import React from 'react';
-import { storybookArgTypes } from '@lg-tools/storybook-utils';
-import { StoryFn } from '@storybook/react';
+import { StoryMetaType, StoryType } from '@lg-tools/storybook-utils';
 
 import { CardSkeleton } from '..';
 
 export default {
   title: 'Components/SkeletonLoader',
   component: CardSkeleton,
+  parameters: {
+    default: null,
+    controls: { exclude: ['darkMode', 'ref'] },
+    generate: {
+      storyNames: ['Card'],
+      combineArgs: {
+        darkMode: [false, true],
+      },
+      decorator: Instance => (
+        <div style={{ width: 500 }}>
+          <Instance />
+        </div>
+      ),
+    },
+  },
+  args: {
+    enableAnimations: false,
```

Comment:
Should this be true by default?

Reply:
Not for the snapshot stories. We want to disable animations here so the "shimmer" effect stops, which ensures consistent UI snapshots